src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContent.java

```diff
@@ -140,30 +136,15 @@ public class NoticeOfProceedingContent {
     @Autowired
     private DocmosisTemplatesConfig config;
 
-    @Value("${court.locations.serviceCentre.serviceCentreName}")
```

Comment:
I see a few more of these have been removed (centre name, service centre and phone number) do these need to be added to DocmosisCommonContent for the template to still work?